book/networking/p2p/README.md

```diff
@@ -0,0 +1,5 @@
+
+- Link to [devp2p](https://github.com/ethereum/devp2p)
+- Should include a brief overview of each p2p subprotocol
+    - (Important) message payloads for each
```

Comment:
a brief summary of the subprotocols should be sufficient here, and link to the spec instead.

Reply:
considering the scope that this chapter has grown to, going to save this documentation for a separate issue/pr